src/main.rs

```diff
@@ -146,6 +155,25 @@ fn main() {
         process::exit(0);
     }
 
+    // Set any global constants requested by the user. This should be the ONLY
+    // block of code that mutates these values.
+    let mut matcher_config = matchers::Config::new();
```

Comment:
This is a really long variable name: can we find a nicer shorter one? e.g. something like just `cfg`?